### PR TITLE
Add SKYSTARSF7RS

### PR DESCRIPTION
**Pull request by freasy**

### CODE REVIEW

**Comment by haslinghuis** [comment]
We already have the SKYSTARSF7HDPRO target.

File: configs/SKYSTARSF7RS/config.h [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        SKYSTARSF7HDPRO
```
Comment (blckmn) [inline configs/SKYSTARSF7RS/config.h:26]: Board name and directory name should match.

**Review by blckmn — CHANGES_REQUESTED** [review]

File: configs/SKYSTARSF7RS/config.h [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+#define BARO_SPI_INSTANCE SPI2
+
+#define USE_ADC
+#define ADC_INSTANCE ADC2
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 290
+#define BEEPER_INVERTED
+#define DEFAULT_PID_PROCESS_DENOM 1
```
Comment (haslinghuis) [inline configs/SKYSTARSF7RS/config.h:109]: setting can be removed - as it is default
```suggestion
#define DEFAULT_PID_PROCESS_DENOM 1
```

File: configs/SKYSTARSF7RS/config.h [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+//TODO #define OSD_RSSI_POS 2445
+//TODO #define OSD_TIM_2_POS 2133
+//TODO #define OSD_CROSSHAIRS_POS 2285
+//TODO #define OSD_CURRENT_POS 2156
+//TODO #define OSD_WARNINGS_POS 14761
+//TODO #define OSD_AVG_CELL_VOLTAGE_POS 2114
```
Comment (haslinghuis) [inline configs/SKYSTARSF7RS/config.h:115]: Remove user settings

**Comment by haslinghuis** [comment]
@freasy can you please also create an unified target for this.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**